src/d3a/d3a_core/simulation.py

```diff
@@ -303,7 +303,6 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
 
         config.external_redis_communicator.sub_to_aggregator()
         config.external_redis_communicator.start_communication()
-        self._update_and_send_results()
```

Comment:
Can you help me understand the intuition behind it?

Reply:
This is redundant because between the call of `_update_and_send_results` in `_init` and inside the slot nothing happened to the results (no updates or at least none that the d3a-web is interested in.